evergreen.yml

```diff
@@ -399,15 +413,32 @@ functions:
   # Requires f_compile to have been run first.
   ##
   f_resmoke_test:
-  - command: subprocess.exec
+  - command: shell.exec
     params:
       continue_on_err: true
+      working_dir: src
+      shell: bash
+      script: |
+        set -o errexit
+        set -o pipefail
+        set -o nounset
+
+        # for python3
+        export PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+
+        ./scripts/lamp resmoke-test ${resmoke_test_args}
```

Comment:
1. can we make `lamp` smart enough to know about the toolchain rather than copy/pasting the `export PATH` in all the places we call it?
2. also I think `set -eou pipefail` is good enough for the first 3 lines

with both of these this block becomes just 2 lines which would help keep our evergreen yaml burden down.